drivers/serial/stm32f1-uart.c

```diff
@@ -67,7 +67,7 @@ static struct inode stm32f1_inode = {
     .i_private = &stm32f1_uart2,
 };
 
-int stm32f1_init(void)
+int serial_init(void)
```

Comment:
If certain platform consists of two UART implementations, the renaming might cause conflicts.

Reply:
Yep, that's a potential problem. But in the view of `serial_init()`, that's not its responsibility to take care of this. It should be done by platform-specified function, such as `plat_usart_init()`. However, it's just a step for refactoring progress, if the reviewer thinks it would make a confused, I will meld this previous commits into one which fixed this problem.

Reply:
Don't get people confused with the naming. No conflicts!